XmlFormatter/src/Manager/VersionManager.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Octokit;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using XmlFormatter.src.DataContainer;
+using XmlFormatter.src.EventMessages;
+
+namespace XmlFormatter.src.Manager
+{
+    class VersionManager
+    {
+        public event EventHandler<BaseEventArgs> Error;
+
+        /// <summary>
+        /// This method will return you the current version of the application
+        /// </summary>
+        /// <returns>The current version of the application</returns>
+        public Version GetApplicationVersion()
+        {
+            Assembly assembly = Assembly.GetExecutingAssembly();
+            string versionString = "0.0.0";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'versionString'.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5002975)